Client/containers/Nav.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { toggleCollectionList, toggleFilterList } from '../actions/index.js';
+import Panel from './Panel';
+
+class Nav extends React.Component {
+
+  collectionClick(e) {
+    e.preventDefault();
+    this.props.toggleCollectionList(this.props.PanelMode('collection'));
```

Comment:
this.props.PanelMode should be a variable containing 'collection', 'filter', or null.  You shouldn't be passing anything into it, as this.props.PanelMode is not a function.